types/decode/decode.d.ts

```diff
@@ -43,3 +43,10 @@ export interface DecodedPayload {
     network?: NetworkEvent[];
     paint?: PaintEvent[];
 }
+
+export interface DecodedVersion {
+    major: number;
+    minor: number;
+    patch: number;
+    beta: number;
```

Comment:
Should change this to be prerelease. See general comment elsewhere about use of semver in this package. Happy to chat.

Reply:
Yes will do prerelease and will chat offline. Thanks.

Reply:
Not changing this but eventually move to more reasonable versioning soon.